custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -292,18 +297,27 @@ def _followups(self):
 
     @property
     def state(self):
+        if self.test_phi is not None:
+            return MockLocation('FAKESTATE', 'fake_state_id', MockLocationType('state', 'state'))
```

Comment:
is there a chance we should use real (as in existing `Location` objects) but fake (as in test or QA locations) data for this instead of mocks? It seems like that would allow us to exercise things that may depend on the hierarchy and make it easier to sync phones.